azure/templates/run-smoke-tests.yml

```diff
@@ -14,7 +14,7 @@ steps:
       export CLIENT_SECRET="not-used-on-smoketest"
       export REDIRECT_URL="not-used-on-smoketest"
 
-      make smoketest
+      poetry run pytest -v tests/api_tests.py --junitxml=tests/api_tests.xml 
```

Comment:
We'll have multiple test files. If you remoe that `tests/api_tests.py` then it'll find test files